tests/features/data-visibility-and-review.feature

```diff
@@ -16,78 +16,107 @@ Feature: Manage data visibility separately from review and approval in the backe
-
-  Scenario: Update review status for a new dataset without changing visibility
-    Given the system has a new dataset with visibility "internal" and review_status "provisional"
-    When an authorized reviewer updates the dataset review_status to "approved" via the api
-    Then the system should persist the dataset with review_status "approved"
-    And the system should preserve the visibility as "internal"
-    And retrieving the dataset via the api should return visibility "internal" and review_status "approved"
 
-  Scenario: Allow all datasets for internal users
-    Given the system has datasets with combinations of:
-      | visibility | review_status |
-      | internal   | provisional   |
-      | internal   | approved      |
-      | public     | provisional   |
-      | public     | approved      |
-    And the caller is identified as an internal staff user
-    When the internal staff user requests those datasets via the api
-    Then the system should return all of those datasets in the response
-
-  Scenario: Include disclaimer information based on review status
-    Given the system has a dataset with visibility "public" and review_status "provisional"
-    When any authorized caller retrieves the dataset via the api
-    Then the system should return a response in JSON format
-    And the response should include a disclaimer field derived from the review_status
-    And the disclaimer should indicate that the data is provisional
+    # Given Require visibility and review status when creating a new record. no default for visibility and review
+      # status will be defined so this is impossible to test
+    And the error response should not include default values for visibility or review_status
 
-    And the system has a dataset with visibility "internal" and review_status "approved"
-    When any authorized caller retrieves that dataset via the api
-    Then the response should include a disclaimer field derived from the review_status
-    And the disclaimer should indicate that the data is approved
+    Examples:
+    |model|
+    |Well |
+    |Contact|
+    |Observation|
 
-  Scenario: Migrate legacy combined visibility/review values to separate fields
-    Given the system has legacy records with a combined visibility/review field
+  Scenario Outline: Migrate legacy combined visibility/review values to separate fields
+    Given the system has legacy records with a combined visibility/review field for <legacy_table>
     And there is a documented mapping from each legacy value to a visibility and review_status pair
-    When the migration job runs to populate visibility and review_status for legacy records
+    When the migration job runs to populate visibility and review_status for <legacy_table> records
     Then the system should set visibility according to the documented mapping
     And the system should set review_status according to the documented mapping
+    # hard to test?

```

Comment:
We can put these mappings into a test and then ensure that the function works correctly in some unit tests